gammapy/region/tests/test_circle.py

```diff
@@ -50,3 +52,11 @@ def test_sky_to_pix_to_sky(self):
         assert_allclose(sky.pos.l, sky2.pos.l)
         assert_allclose(sky.pos.b, sky2.pos.b)
         assert_allclose(sky.radius, sky2.radius)
+
+    def test_area(self):
+        pos = SkyCoord.from_name('crab')
+        rad = Angle('1.3451 deg')
+        reg = SkyCircleRegion(pos, rad)
+        desired = (rad ** 2 * np.pi).to('steradian')
```

Comment:
Leave a comment that this is the small angle approximation, and the method is the spherical cap area.